ods_ci/tests/Resources/Page/Operators/ISVs.resource

```diff
@@ -0,0 +1,114 @@
+*** Settings ***
+Documentation       Collcetion of keywords to manage ISV operators via CLI
+Resource            ../../RHOSi.resource
+Library             OperatingSystem
+
+
+*** Variables ***
+${FILES_RESOURCES_DIRPATH}=            ods_ci/tests/Resources/Files
+${SUBSCRIPTION_YAML_TEMPLATE_FILEPATH}=            ${FILES_RESOURCES_DIRPATH}/isv-operator-subscription.yaml
+${OPERATORGROUP_YAML_TEMPLATE_FILEPATH}=            ${FILES_RESOURCES_DIRPATH}/isv-operator-group.yaml
+
+
+*** Keywords ***
+Install ISV Operator From OperatorHub Via CLI
```

Comment:
hm I looked for a similar kw but didn't find it among the RF resources files. Why `install_openshift_isv` is in ocm.py? 
Anyway, I can try re-using this kw

Reply:
we talked offline and agreed on keep using the RF one since it give some more benefits:
- clear debug in the test report
- easier to find it while coding
- consistent with the rest of kw (e.g., Uninstall of ISV operator)

In Q3 we'll work together to align the test case which is using the python keyword (i.e., sandbox testing) to use the RF keyword + refine the RF to use jinja templates